renku/core/management/workflow/converters/cwl.py

```diff
@@ -361,10 +362,15 @@ def _convert_input(input: CommandInput, basedir: Path):
                 prefix = prefix[:-1]
                 separate = True
 
+            input_binding = cwl.CommandLineBinding(position=position, prefix=prefix, separate=separate)
+
+        if position:
```

Comment:
Should this be `elif`?

Reply:
that was my first ideas as well, but felt cleaner like this. tbh no hard preference about this at all